src/core_modules/capture-core/components/Pages/MainPage/EventWorkingListsCommon/index.js

```diff
@@ -0,0 +1,9 @@
+// @flow
+export { useDefaultColumnConfig } from './defaultColumnConfiguration';
+export type {
+    EventRecords,
+    MainColumnConfig,
+    MetadataColumnConfig,
+    EventWorkingListsColumnConfig,
+    EventWorkingListsColumnConfigs,
+} from './types';
```

Comment:
My idea here is:

- The types folder contains types that are reusable internally in `EventWorkingListsCommon`.

- The index file in the base folder will export the types that are exposed (to other components).

- The reason for adding the index file in the types folder is to able to organize the types in files in whatever way you want and also align this folder with the other folders (which also contains index files)

Might be overkill sometimes, but I like having a pattern to follow, it's cognitively easier.

We can certainly discuss if this is a good pattern. Please add it to the discussion points if so :)

Reply:
I think keeping a pattern to follow for consistency is good. And since you have been doing it for a while now might not be the best to change it. There are more pressing things to work on I believe. 

Perhaps we can document this in our wiki because then I could better understand it and follow it on the code I write. 
---

Now about the code it self could we do `*` here?

Reply:
well, everything shouldn't always be exposed to the outside "world". But in this case it seems to add up so changing to `*`
I added something to the wiki, we can expand on it later (feel free to change / expand on it)